usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -194,6 +194,8 @@ def keyword_parse(keyword):
                     filter_obj = Q(parent_recipient_unique_id=parent_duns)
                 elif len(parent_duns_rows) > 2:
                     raise InvalidParameterException('Non-unique Record in ')
+                else:
+                    raise InvalidParameterException('Parent Recipient ID ({}) not found'.format(recipient_hash))
```

Comment:
This might not be desired behavior. Will need to discuss with team

Reply:
I can take it out if need be. It just led to confusing results when I accidentally listed a recipient as a parent.

Reply:
Confusing as too many or too-few results? My guess is a similar confusion can occur if someone lists the incorrect date period in the filter. I added the test for too many parent records found since that should never happen, if it does there is an issue with the table data.

Reply:
That makes sense and I can take it out for that. It was the case when no results were found and it turned out I was searching for a recipient (level `R`) even though the level in the request was `P`.